source/mainnet/net/guides/run-node-windows.rst

```diff
@@ -20,6 +20,14 @@ Prerequisites
 Install/upgrade and run a node
 ==============================
 
+.. Note::
+
+   Node version 4.5.0 introduced the GRPC V2 interface which is enabled by
+   default. If you have done special configuration of your node and want to
+   re-use the configuration file and have the new API enabled, make sure to edit
+   your configuration and add ``grpc2.port`` and ``grpc2.address`` settings. See
+   :ref:`configure-node` for more information.
+
 .. Note::
```

Comment:
Should we add example?
`grpc2.port = 20000`
`grpc2.ip = "127.0.0.1"
`

Reply:
I was hoping we could rely on the documentation that is linked, but I suppose it is better to just directly tell them.

The only inconvenience is that we need to tell them "use this for testnet" and "this for mainnet"